modules/addBooks.js

```diff
@@ -0,0 +1,15 @@
+import Book from './books.js';
+
+export default function addBooks(event) {
```

Comment:
- [ ] Please use the `arrow syntax` here. We are using ES6 Javascript in this milestone and every function should be an arrow function.